chat-commands.js

```diff
@@ -1138,17 +1135,18 @@ exports.commands = {
 
 		room.staffMessage = target.replace(/\r/g, '');
 		this.sendReply("(The staff introduction has been changed to:)");
-		this.sendReply('|raw|<div class="infobox">' + target.replace(/\n/g, '') + '</div>');
+		this.sendReply(`|raw|<div class="infobox">${target.replace(/\n/g, ``)}</div>`);
 
 		this.privateModAction(`(${user.name} changed the staffintro.)`);
 		this.modlog('STAFFINTRO');
-		this.roomlog(room.staffMessage.replace(/\n/g, ''));
+		this.roomlog(room.staffMessage.replace(/\n/g, ``));
 
 		if (room.chatRoomData) {
 			room.chatRoomData.staffMessage = room.staffMessage;
 			Rooms.global.writeChatRoomData();
 		}
 	},
+
```

Comment:
If you're adding a line here, then all commands should have a line between them, too... (see line 1095)

Reply:
Done